src/main/java/com/timelogr/enterprise/client.java

```diff
@@ -1,51 +0,0 @@
-package com.timelogr.enterprise;
-
-public class client {
-    private long id;
-    private String name;
-    private String email;
-    private String number;
-
-    public client() {
-        super();
-    }
-
-    public client(long id, String name, String email, String number) {
-        this.id = id;
-        this.name = name;
-        this.email = email;
-        this.number = number;
-    }
-
-    public long getId() {
-        return id;
-    }
-
-    public void setId(long id) {
-        this.id = id;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    public String getEmail() {
-        return email;
-    }
-
-    public void setEmail(String email) {
-        this.email = email;
-    }
-
-    public String getNumber() {
-        return number;
-    }
-
-    public void setNumber(String number) {
-        this.number = number;
-    }
-}
```

Comment:
This is for all "deleted" classes. I simply changed the file name to start with an upper-case letter, following Java naming conventions: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html